task.sql

```diff
@@ -1,11 +1,19 @@
 -- Use our database
 USE ShopDB; 
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Orders (CustomerID, Date)
+VALUES (1, CURRENT_DATE);
+
+SET @order_id = LAST_INSERT_ID();
 
 -- Start the transaction 
-START TRANSACTION; 
+START TRANSACTION;
```

Comment:
The transaction is started here, but the order insertion is already executed before this point. Consider moving the `START TRANSACTION` statement before the `INSERT INTO Orders` statement if you want the entire process to be atomic.